### PR TITLE
refactor: remove deprecated fields in postinfo

### PR DESCRIPTION
**Pull request by Himself65**
/cc @Jack-Works

### CODE REVIEW

File: packages/maskbook/src/social-network/defaults/inject/CommentBox.tsx [hunk 1]
```diff
@@ -28,20 +29,22 @@ export const injectCommentBoxDefaultFactory = function <T extends string>(
...
+        const postContent = usePostInfoDetails('transformedPostContent')
         const styles = useCustomStyles()
         const iv = usePostInfoDetails('iv')
         const props = additionPropsToCommentBox(styles)
         const onCallback = useCallback(
             async (content) => {
                 const postIV = iv || payload.unwrap().iv
-                const encryptedComment = await Services.Crypto.encryptComment(postIV, decrypted, content)
+                // TODO: currently we only one item which is TypedMessageText in the tuple
+                const decryptedText = (postContent.items[0] as TypedMessageText).content
```
Comment (Jack-Works) [inline packages/maskbook/src/social-network/defaults/inject/CommentBox.tsx:40]: IMO you should use isTypedMessageText

Comment (Himself65) [reply packages/maskbook/src/social-network/defaults/inject/CommentBox.tsx:40]: how?

File: packages/maskbook/src/components/InjectedComponents/PostComments.tsx [hunk 2]
```diff
@@ -45,17 +47,24 @@ export interface PostCommentProps {
...
     const dec = useAsync(async () => {
-        if (!postIV || !decryptedPostContent) throw new Error('Decrypt comment failed')
-        const result = await Services.Crypto.decryptComment(postIV, decryptedPostContent, comment)
+        // TODO: currently we only one item which is TypedMessageText in the tuple
+        let decryptedText: string
+        if (isTypedMessageText(postContent.items[0])) {
+            decryptedText = (postContent.items[0] as TypedMessageText).content
+        } else {
+            throw new Error('Expected TypedMessageText')
+        }
```
Comment (Jack-Works) [inline packages/maskbook/src/components/InjectedComponents/PostComments.tsx:62]: Oh I'm sorry in this case you should use `extractTextFromTypedMessage`

**Review by Jack-Works — APPROVED** [review]

**Comment by Himself65** [comment]
What’s more, I’m working on removing other duplicated fields.

**Pull request merged**